deploy/restartComponent.sh

```diff
@@ -3,16 +3,17 @@
 HOST=`hostname`
 DATENOW=`date +%s`
 
+# Get a few environment variables in, like $install and $manage
+source /data/admin/wmagent/env.sh
+
 # Figure whether it's a python2 or python3 agent
-if [ -d "/data/srv/wmagent/current/install/wmagent" ]; then
-  WMA_PATH="/data/srv/wmagent/current/install/wmagent"
-else
-  WMA_PATH="/data/srv/wmagent/current/install/wmagentpy3"
+if [ ! -d "$install" ]; then
```

Comment:
This variable `$install` is not set anywhere from inside this script, but is checked  if not provided by the environment.  I'd add a comment line mentioning who/which component is expected to set it.

Reply:
Did you miss my changes in line 6 and 7?